src/test/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumerIntegrationDefaultModeTest.java

```diff
@@ -0,0 +1,157 @@
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@DirtiesContext
+@EmbeddedKafka
+@TestPropertySource(properties="uk.gov.companieshouse.item-handler.error-consumer=false")
+public class OrdersKafkaConsumerIntegrationDefaultModeTest {
+    private static final String ORDER_RECEIVED_TOPIC = "order-received";
+    private static final String ORDER_RECEIVED_TOPIC_RETRY = "order-received-retry";
+    private static final String ORDER_RECEIVED_TOPIC_ERROR = "order-received-error";
+    private static final String GROUP_NAME = "order-received-consumers";
+    private static final String ORDER_RECEIVED_URI = "/order/ORDER-12345";
+    @Value("${kafka.broker.addresses}")
+    private String brokerAddresses;
+
+    private KafkaTemplate<String, String> template;
+
+    private KafkaMessageListenerContainer<String, String> container;
+
+    private BlockingQueue<ConsumerRecord<String, String>> records;
+
+    @Autowired
+    private OrdersKafkaConsumerWrapper consumerWrapper;
+
+    @Before
+    public void setUp() {
+        setUpTestKafkaOrdersProducer();
+        setUpTestKafkaOrdersConsumer();
+    }
+
+    @After
+    public void tearDown() {
+        container.stop();
+    }
+
+    private void setUpTestKafkaOrdersProducer() {
+        final Map<String, Object> senderProperties = KafkaTestUtils.senderProps(brokerAddresses);
+
+        senderProperties.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
+
+        final ProducerFactory<String, String> producerFactory = new DefaultKafkaProducerFactory<>(senderProperties);
+
+        template = new KafkaTemplate<>(producerFactory);
+        template.setDefaultTopic(ORDER_RECEIVED_TOPIC);
+    }
+
+    private void setUpTestKafkaOrdersConsumer() {
+        final Map<String, Object> consumerProperties = new HashMap<>();
+        consumerProperties.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        consumerProperties.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        consumerProperties.put(ConsumerConfig.GROUP_ID_CONFIG, GROUP_NAME);
+        consumerProperties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerAddresses);
+
+        final DefaultKafkaConsumerFactory<String, String> consumerFactory =
+                new DefaultKafkaConsumerFactory<>(consumerProperties);
+
+        final ContainerProperties containerProperties = new ContainerProperties(ORDER_RECEIVED_TOPIC);
+
+        container = new KafkaMessageListenerContainer<>(consumerFactory, containerProperties);
+
+        records = new LinkedBlockingQueue<>();
+
+        container.setupMessageListener((MessageListener<String, String>) record -> {
+                    records.add(record);
+                });
+
+        container.start();
+
+        ContainerTestUtils.waitForAssignment(container, 0);
+    }
+
+    @Test
+    public void testOrdersConsumerReceivesOrderReceivedMessage() throws InterruptedException, ExecutionException {
```

Comment:
I think the names of the tests in these two files could be changed to better reflect what they verify. `@DisplayName` might be useful here too.

Reply:
Updated